adoptopenjdk-api-v3-frontend/pom.xml

```diff
@@ -151,29 +151,6 @@
         <sourceDirectory>src/main/kotlin</sourceDirectory>
         <testSourceDirectory>src/test/kotlin</testSourceDirectory>
         <plugins>
-            <plugin>
```

Comment:
The default artifact contains now all the compiled classes and resources, that's why attach-artifact shouldn't be required anymore and the jar-plugin is in again.